components/hacker/HackerDash.tsx

```diff
@@ -93,16 +93,19 @@ export default function HackerDash({ userApplicationStatus, setUserApplicationSt
 	);
 
 	// get school options
-	const [schoolOptions, setSchoolOptions] = useState([]);
+	const [schoolOptions, setSchoolOptions] = useState<DropdownItem[]>([]);
 	const [school, setSchool] = useState('');
 	const { data: schools } = useSWR(
-		'http://universities.hipolabs.com/search',
+		'https://raw.githubusercontent.com/MLH/mlh-policies/main/schools.csv',
 		async url => {
 			const res = await fetch(url, { method: 'GET' });
-			const jsonRes = await res.json();
-			const schoolList = jsonRes.map((school: any) => {
-				return { label: school.name, value: school.name };
+			const csvText = await res.text();
+			const schoolArr = csvText.split('\n').slice(1);
+
+			const schoolList = schoolArr.map((school: any) => {
```

Comment:
what if and just hear me out there was a type here

Reply:
what if and hear me out u had a crush on me

Reply:
fine but just know im punching u in my head!